at_client/src/test/java/org/atsign/virtualenv/VirtualEnv.java

```diff
@@ -31,7 +31,7 @@ public static void main(String[] args) throws Exception {
   public static void setUp() {
     try {
       URL resource = VirtualEnv.class.getResource("docker-compose.yml");
-      StartUpLatch latch = new StartUpLatch(Pattern.compile("install_PKAM_Keys .*successful"), 10);
+      StartUpLatch latch = new StartUpLatch(Pattern.compile("install_PKAM_Keys .*successful"), 40);
```

Comment:
Checks were (sometimes) failing because we weren't waiting for all of the atSigns in the virtual environment to have their pkam public keys stored